app/controllers/claims/support/schools/claims_controller.rb

```diff
@@ -54,12 +54,16 @@ def update
   end
 
   def draft
+    return redirect_to reject_claims_support_school_claim_path if @claim.invalid_mentor_training_hours?
+
     success_message = @claim.draft? ? t(".update_success") : t(".add_success")
     Claims::Claim::CreateDraft.call(claim: @claim)
 
     redirect_to claims_support_school_claims_path(@school), flash: { success: success_message }
   end
 
+  def reject; end
```

Comment:
This reads like a user-triggered action to me. Like, "I want to reject _thing_".

Can we rename this to `rejected` please? This way it reads like feedback to the user.